app/src/main/res/layout/item_transaction_perk_bonus.xml

```diff
@@ -19,48 +19,56 @@
       app:layout_constraintTop_toTopOf="parent"
       tools:src="@drawable/transactions_promotion_bonus"
       />
-
-  <TextView
-      android:id="@+id/bonus_title"
-      android:layout_width="wrap_content"
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
We don't need to add a constraint layout to align the texts and give the proper margin to them